### DISCUSSION TITLE
Can we do anything to help 2.8.3 release?

### DISCUSSION THREAD

**Discussion by ippeiukai** [chakra-ui/chakra-ui#8853] [General] [answered]
I really want to update my product to use latest chakra v2, but earlier issue with `disabled` property is forcing us to stay at `@chakra-ui/react@<2.4.9`. It is not feasible to rewrite every use of `disabled` property in a large repository.

The issue is already fixed on v2 branch. Is there anything we can do to help another v2 release?

- https://github.com/chakra-ui/chakra-ui/issues/7269#issuecomment-2337531540
- https://github.com/chakra-ui/chakra-ui/issues/7965#issuecomment-2249022635

**Comment by segunadebayo** [comment, marked as answer]
This is now done ✅

**Comment by ippeiukai** [comment]
Thanks!
